setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='orbeon-xml-api',
-    version='0.1.0.dev28',
+    version='0.1.0.dev23',
```

Comment:
Please increment to version `0.1.0.dev29`, so we won't break existing configurations/installations.